app/components/Switch.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Switch as RNSwitch } from 'react-native';
+
+import Colors from '../constants/colors';
+
+export const Switch = ({
+  onValueChange,
+  value = false,
+  trackColor = { false: Colors.GRAY_BACKGROUND, true: Colors.SUCCESS },
```

Comment:
`// TODO(https://pathcheck.atlassian.net/browse/SAF-230): make theme aware`

prefer to tackle this using styled components and `({theme}) => theme.success` and `theme.danger`, but better for another PR

Something like: 
- `<Switch>` (green track when true) 
- `<Switch danger>` (red track when true) 
- `<Switch danger invert>` (red when false)

Reply:
thanks for making that ticket.